src/incentives/package.json

```diff
@@ -1,59 +1,60 @@
 {
-    "name": "incetives-service",
-    "version": "0.0.0",
-    "private": true,
-    "main": "node ./bin/www",
-    "scripts": {
-        "start": "NODE_ENV=production node ./bin/www",
-        "dev-mac": "NODE_ENV=development nodemon ./bin/www",
-        "prod-mac": "NODE_ENV=production nodemon ./bin/www",
-        "stage-mac": "NODE_ENV=staging nodemon ./bin/www",
-        "dev-pc": "SET NODE_ENV=development&&nodemon ./bin/www",
-        "prod-pc": "SET NODE_ENV=production&&nodemon ./bin/www",
-        "stage-pc": "SET NODE_ENV=staging&&nodemon ./bin/www",
-        "prettier": "prettier --single-quote --print-width 80 --trailing-comma all --write 'src/**/*.js'",
-        "test": "mocha ./**/test/ut_*.js --exit"
-    },
-    "_moduleAliases": {
-        "@root": ".",
-        "@models": "models/",
-        "@utils": "utils",
-        "@controllers": "controllers",
-        "@config": "config",
-        "@middleware": "middleware",
-        "@scripts": "scripts",
-        "@routes": "routes"
-    },
-    "dependencies": {
-        "@log4js-node/slack": "^1.0.0",
-        "body-parser": "~1.18.2",
-        "compression": "^1.7.4",
-        "cookie-parser": "~1.4.3",
-        "debug": "~2.6.9",
-        "dotenv": "^8.2.0",
-        "express": "^4.17.1",
-        "express-validator": "^6.12.2",
-        "google-libphonenumber": "^3.2.32",
-        "helmet": "^3.21.1",
-        "http-status": "^1.5.0",
-        "is-empty": "^1.2.0",
-        "log4js": "^6.3.0",
-        "module-alias": "^2.2.3",
-        "mongoose": "^5.13.7",
-        "mongoose-unique-validator": "^2.0.3",
-        "morgan": "~1.9.0",
-        "mtn-momo": "^2.0.0",
-        "node-fetch": "^2.6.1",
-        "nodemon": "^1.19.4",
-        "request": "^2.88.0"
-    },
-    "devDependencies": {
-        "chai": "^4.3.7",
-        "chai-http": "^4.4.0",
-        "istanbul": "^0.4.5",
-        "mocha": "^10.2.0",
-        "sinon": "^15.2.0",
-        "sinon-chai": "^3.7.0",
-        "supertest": "^6.3.3"
-    }
+  "name": "incetives-service",
+  "version": "0.0.0",
+  "private": true,
+  "main": "node ./bin/www",
+  "scripts": {
+    "start": "NODE_ENV=production node ./bin/www",
+    "stage": "NODE_ENV=staging node ./bin/www",
```

Comment:
[add missing 'stage' script](https://github.com/airqo-platform/AirQo-api/pull/1875/commits/44d89f1d2ae756f3d1ce8911f558cf322a2fb987) cc @Baalmart 
The rest of the changes were indenting ones by the linter.